HEC.FDA.ViewModel/Utilities/ImportFromFDA1Helper.cs

```diff
@@ -715,6 +715,15 @@ private static ContinuousDistribution ConvertRatioValueToFDA2(ContinuousDistribu
                     double newLogStDev = logMean * logStDev / 100;
                     retval = new LogNormal(logMean, newLogStDev);
                     break;
+                case IDistributionEnum.Triangular:
```

Comment:
does it make sense to **rename these variables** to remind us that the min that we import is actually min as a percent of most likely, and that max is max as a percent of most likely?

Reply:
Comments here could and should be better as a whole. i will make some edits